tests/src/test/scala/mhtml/tests.scala

```diff
@@ -325,9 +325,10 @@ class Tests extends FunSuite {
     assert((104, 105) == result)
   }
 
-  test("unsafeRawHTML") {
+  test("Use mhtml-onmount to set .innerHTML") {
```

Comment:
Can we add a test to imitate the d3 behavior? I'm not sure exactly how it works, but I think it's something like this

```scala
  test("Use mhtml-onmount to set .innerHTML") {
    val mutateMe = dom.createElement("h1")
    mutateMe.innerHtml = "foobar"
    val ra = va.map(a => <div mhtml-onmount={ e => e.appendChild(mutateMe) }></div>)
    val div = dom.document.createElement("div")
    mount(div, ra)
    assert(div.innerHTML == "<div><h1>foobar</h1></div>")
    mutateMe.appendChild(dom.createElement("h2"))
    assert(div.innerHTML == "<div><h1>foobar<h2></h2><br></h1></div>")
}
```